actions/hdl_example/sw/snap_example.c

```diff
@@ -222,6 +222,18 @@ static void action_memcpy(struct snap_card* h,
 	return;
 }
 
+static int memcmp2(uint8_t *src, uint8_t *dest, int len)
+{
+	int i;
+
+	for (i = 0; i < len; i++) {
+		if (*src != *dest)
+			return i;
```

Comment:
Hm, I guess you did not want to return 0 if a difference occurs immediately in the first position?